devicetypes/heltun/heltun-ft01-fan-coil-thermostat.src/heltun-ft01-fan-coil-thermostat.groovy

```diff
@@ -0,0 +1,569 @@
+/**
+ *  HELTUN FT01 Fan Coil Thermostat
+ *
+ *  Copyright 2021 Sarkis Kabrailian
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ *  in compliance with the License. You may obtain a copy of the License at:
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software distributed under the License is distributed
+ *  on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License
+ *  for the specific language governing permissions and limitations under the License.
+ */
+ 
+
+metadata {
+	definition (name: "HELTUN FT01 Fan Coil Thermostat", namespace: "HELTUN", author: "Sarkis Kabrailian", cstHandler: true, ocfDeviceType: "oic.d.thermostat") {
+		capability "Energy Meter"
+		capability "Fan Speed"
+		capability "Power Meter"
+		capability "Relative Humidity Measurement"
+		capability "Temperature Measurement"
+		capability "Thermostat Heating Setpoint"
+		capability "Thermostat Mode"
+		capability "Thermostat Operating State"
+		capability "Illuminance Measurement"
+		capability "Configuration"
+		capability "Health Check"
+		capability "Refresh"
+
+		attribute "thermostatFanState", "string"
```

Comment:
custom attributes are not supported (without any custom capability) and this one will not work here.